src/core/component/reflect/constructor.ts

```diff
@@ -67,6 +67,8 @@ export function getInfoFromConstructor(
 	const
 		name = declParams?.name ?? getComponentName(constructor);
 
+	const layer = declParams?.layer ?? undefined;
```

Comment:
Мне кажется, что здесь лишний оператор `??`. `layer` ведь и так будет undefined, потому:

```suggestion
	const layer = declParams?.layer;
```